code/modules/mob/emote.dm

```diff
@@ -1,3 +1,21 @@
+#define EMOTE_COOLDOWN 30		//Time in deciseconds that the cooldown lasts
+
+//Emote Cooldown System
+/mob/proc/handle_emote_CD(cooldown = EMOTE_COOLDOWN)
+	if(emote_cd == 3) //Spam those emotes
+		return FALSE
+	if(emote_cd == 2) // Cooldown emotes were disabled by an admin, prevent use
+		return TRUE
+	if(emote_cd == 1)  // Already on CD, prevent use
+		return TRUE
+
+	emote_cd = TRUE	// Starting cooldown
+	spawn(cooldown)
```

Comment:
spawn usage is banned 
find another way
i'd recommend doing last_sound_emote or next_sound_emote var on client, or at the very least an addtimer